packages/selinux-policy/rules.cil

```diff
@@ -246,6 +246,7 @@
 (allow clock_s measure_t (files (mutate)))
```

Comment:
For the lack of a better place to put this, your commit message refers to a label that doesn't exist (`process_s`). I think you meant `trusted_s`.